app/lib/forward.rb

```diff
@@ -0,0 +1,69 @@
+require 'line/bot'
+
+class Forward
+  class << self
+    def talk(token, message)
+      initiate_client
+      @client.reply_message(token, forwardToCofacts(message))
+    end
+
+    private
+
+    def initiate_client
+      @client = Line::Bot::Client.new do |config|
+        config.channel_secret = ENV['line_channel_secret']
+        config.channel_token = ENV['line_channel_token']
+      end
+    end
+
+    def forwardToCofacts message
```

Comment:
ruby style method name with underscore (not necessary)

```suggestion
    def forward_to_cofacts message
```